src/main/java/org/usfirst/frc4904/robot/commands/LowerHook.java

```diff
@@ -0,0 +1,39 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class LowerHook extends CommandBase {
+  public static final double DEFAULT_DOWN_SPEED = -0; // TODO: This is an untested value
+  protected final Motor tubeMotor;
+  protected double speed;
+
+  /**
+   * Lower the deploy tube to hook onto the generator switch
+   * 
+   * @param tubeMotor The motor to use to lower
+   * @param speed     The speed to lower the deploy tube at (should be a positive
+   *                  number)
```

Comment:
Why should this be a positive number? While good in theory, this feels like the kind of thing that could be confusing while testing. Also, this assumes the motor polarity.